generatedComponentConfig.js

```diff
@@ -80,7 +82,24 @@ const componentConfig = {
   'terra-clinical-action-header': {
     name: 'Clinical Action Header',
     path: '/clinical-action-header',
+    pages: [
+      {
+        name: 'Index',
+        path: '/index',
+        component: TerraClinicalActionHeaderIndex,
+      },
+    ],
     tests: [
+      {
+        name: 'Action Header Test Routes',
+        path: '/action-header-test-routes',
+        component: ActionHeaderTestRoutes,
```

Comment:
The ActionHeaderTestRoutes & ActionHeaderTests files can be removed. These were the hand-made route files that we no longer need. See #275. I did not remove them from the PR because it would have increased the number of files to reviews by 2 x n components.